Unix/http/httpclientauth.c

```diff
@@ -461,7 +461,586 @@ static int EncodePlaceCallback(const char *data, size_t size, void *callbackData
+                {
+                    // Content-Type: application/HTTP-SPNEGO-session-encrypted | Content-Type: application/octet-stream
+
+                    // Scan to the end of the line
+                    while (!('\n' == scanp[0] && '\r' == scanp[-1]) && scanp < scanlimit && !done)
+                        scanp++;
+
+                    linelimit = scanp - 1;
+
+                    linep += CONTENT_TYPE_LEN;
+                    while (isspace(*linep) && linep < linelimit)
+                        linep++;
+
+                    if (':' == *linep && linep < linelimit)
+                        linep++;
+
+                    while (isspace(*linep) && linep < linelimit)
+                        linep++;
+
+                    if (Strncasecmp(linep, OCTET_STREAM, OCTET_STREAM_LEN) == 0)
+                    {
+
+                        sig_len = *(uint32_t *) (linelimit + 2);
+                        sig_len = ByteSwapToWindows32(sig_len);
+
+                        input_buffer.length = original_content_length+sig_len;
+                        input_buffer.value = linelimit + 2  // skip crlf
+                                                       + 4; // skip signature len
+
+                        done = TRUE;
+                        break;
+                    }
+                    else if (Strncasecmp(linep, APPLICATION_SPNEGO, APPLICATION_SPNEGO_LEN) == 0)
+                    {
+                        // Should be application/HTTP-SPNEGO-session-encrypted
+
+                    }
+                    else
+                    {
+                        // Bogus.
+                    }
+                    scanp = linelimit + 2;
+                }
+                else if (Strncasecmp(linep, ORIGINAL_CONTENT, ORIGINAL_CONTENT_LEN) == 0)
+                {
+                    while (!('\n' == scanp[0] && '\r' == scanp[-1]) && scanp < scanlimit && !done)
+                        scanp++;
+
+                    linelimit = scanp - 1;
+
+                    linep += ORIGINAL_CONTENT_LEN;
+                    do
+                    {
+
+                        while ((isspace(*linep) || ';' == *linep) && linep < linelimit)
+                            linep++;
+                        if (Strncasecmp(linep, LENGTH_FIELD, LENGTH_FIELD_LEN) == 0)
+                        {
+                            linep += LENGTH_FIELD_LEN;
+                            original_content_length = atoi(linep);
+                            while (';' != *linep && *linep && linep < linelimit)
+                                linep++;
+                            linep++;
+                        }
+                        else if (Strncasecmp(linep, TYPE_FIELD, TYPE_FIELD_LEN) == 0)
+                        {
+                            linep += TYPE_FIELD_LEN;
+                            original_content_type = linep;
+                            while (';' != *linep && *linep && linep < linelimit)
```

Comment:
Coding standards calls for braces, even for one-line `if` or `while` type statements.